src/sbd.sysconfig

```diff
@@ -23,14 +23,22 @@ SBD_PACEMAKER=yes
 #
 SBD_STARTMODE=always
 
-## Type: yesno
+## Type: yesno / integer
 ## Default: no
 #
 # Whether to delay after starting sbd on boot for "msgwait" seconds.
 # This may be necessary if your cluster nodes reboot so fast that the
 # other nodes are still waiting in the fence acknowledgement phase.
 # This is an occasional issue with virtual machines.
 #
+# This can be either enabled by configuring a specific delay value, in
+# seconds, if a different delay than the default, "msgwait", is needed.
+# For example for the cases where it's considered to be safe enough to
+# wait longer than:
```

Comment:
Not sure here but I guess it should express that a longer delay is sometimes needed than the default.
Then I would expect something like "... to be safer to ...".

Reply:
Indeed this is better, Thanks. Changed. .